### PR TITLE
Partial reorganization of Sensory presentation under attribute

### PR DESCRIPTION
**Pull request by VisLab**
This is a start on suggestion issue #44 of restructuring the Sensory presentation category under attribute.  It is too big a job to do in one pass.  I also implemented Scott's suggestion about on issue #22.

### CODE REVIEW

File: HEDremap.tsv [hunk 1]
```diff
@@ -128,3 +126,45 @@ Attribute/Size/Height/#	Attribute/Spatial/Size/Height/#
...
+Sensory presentation/Visual/Rendering type/Screen/Movie/Motion-capture/Point light	Attribute/Sensory presentation/Visual/Rendering type/Screen/Movie/Motion-capture/Point light
+Sensory presentation/Visual/Rendering type/Screen/Movie/Motion-capture/Stick figure	Attribute/Sensory presentation/Visual/Rendering type/Screen/Movie/Motion-capture/Stick figure
+Sensory presentation/Visual/Rendering type/Screen/Movie/Motion-capture/Outline	Attribute/Sensory presentation/Visual/Rendering type/Screen/Movie/Motion-capture/Outline
+Sensory presentation/Visual/Rendering type/Screen/Movie/Flickering	Attribute/Sensory presentation/Visual/Rendering type/Screen/Movie/Flickering
+Sensory presentation/Visual/Rendering type/Screen/Movie/Steady state	Attribute/Sensory presentation/Visual/Rendering type/Screen/Movie/Steady state
+Sensory presentation/Visual/Rendering type/LED	Attribute/Sensory presentation/Visual/Rendering type/LED
+
+____________________________________________________________________________________
+Sensory presentation/Visual	Attribute/Visual
+Senory presentation/Visual/Real-world
```
Comment (happy5214) [inline HEDremap.tsv]: `Senory` -> `Sensory`.

File: HED-schema.mediawiki [hunk 2]
```diff
@@ -137,16 +148,12 @@ From version 2.2, HED adheres to http://semver.org/ versioning.
 * IAPS <nowiki>[International Affective Picture System]  </nowiki> 
 * IADS <nowiki> [International Affective Digital Sounds]</nowiki> 
 * SAM <nowiki> [The Self-Assessment Manikin]</nowiki> 
+** Music
```
Comment (happy5214) [inline HED-schema.mediawiki]: Is this supposed to be under SAM?

File: HED-schema.mediawiki [hunk 3]
```diff
@@ -360,6 +387,8 @@ From version 2.2, HED adheres to http://semver.org/ versioning.
 
 * Social <nowiki>[Involving interactions among multiple agents such as humans or dogs or robots]</nowiki> 
 * Peak <nowiki>[Peak velocity or  acceleration or jerk]</nowiki>
+* Visual
```
Comment (happy5214) [inline HED-schema.mediawiki]: Visual already exists on line 386/415.

File: HED-schema.mediawiki [hunk 4]
```diff
@@ -195,10 +196,6 @@ From version 2.2, HED adheres to http://semver.org/ versioning.
 *** Yawn 
 ** Nonvocal <nowiki> [A car engine or gears grinding --- anything that is not made by a human or an animal]</nowiki> 
 *** Engine 
-* Olfactory <nowiki>[Odor]</nowiki> 
-* Taste 
-* Tactile <nowiki>[Pressure]</nowiki> 
-* Visual  
 ** Rendering type <nowiki>{requireChild, predicateType=passThrough}</nowiki> 
 *** Screen 
 **** Head-mounted display
```
Comment (dungscout96) [inline HED-schema.mediawiki]: These should be removed as they are being moved under Attribute

File: HED-schema.mediawiki [hunk 5]
```diff
@@ -137,16 +148,12 @@ From version 2.2, HED adheres to http://semver.org/ versioning.
...
 '''Sensory presentation''' <nowiki>{extensionAllowed} [Object manifestation]</nowiki> 
 
 * Auditory <nowiki>[Sound]</nowiki> 
-** Nameable 
-** Cash register  
-** Ding <nowiki>[Often associated with positive valence]</nowiki> 
-** Buzz <nowiki>[Often associated with negative valence]</nowiki> 
-** Fire alarm 
-** Click 
 *** ABR <nowiki>[Auditory Brainstem Response]</nowiki>
```
Comment (dungscout96) [inline HED-schema.mediawiki]: ABR a brain response, not an auditory presentation so I think we should remove this

File: HED-schema.mediawiki (hunk 3 above)
Comment (dungscout96) [reply HED-schema.mediawiki]: I agree

File: HED-schema.mediawiki (hunk 4 above)
Comment (VisLab) [reply HED-schema.mediawiki]: Modified pull request.

File: HED-schema.mediawiki (hunk 5 above)
Comment (VisLab) [reply HED-schema.mediawiki]: I don't know where to put this so I am leaving temporarily in place for next round and further discussion.

File: HED-schema.mediawiki (hunk 2 above)
Comment (VisLab) [reply HED-schema.mediawiki]: No SAM is a set of images (like the pain smiley faces) for assessing mode.  However, I am going to move all under multimedia in this round

**Pull request merged**